NetKAN/Arkas-Config-Med-Clouds.netkan

```diff
@@ -0,0 +1,21 @@
+{
+	"spec_version"	:	1,
+	"identifier"	:	"Arkas-Config-Med-Clouds",
```

Comment:
And here I thought I had done such a good job.

OK, I've implemented all the changes in all 4 files, and renamed the 3 
config files to match the new identifier

Thanks


JBB

On 10/30/2015 3:40 AM, Matthew wrote:
>
> In NetKAN/Arkas-Config-Med-Clouds.netkan 
> <https://github.com/KSP-CKAN/NetKAN/pull/2505#discussion_r43478027>:
>
> > @@ -0,0 +1,21 @@
> > +{
> > +	"spec_version"	:	1,
> > +	"identifier"	:	"Arkas-Config-Med-Clouds",
>
> |Arkas-Clouds-Medium| would be a more descriptive ID and help with 
> location and sorting of the metadata files.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/KSP-CKAN/NetKAN/pull/2505/files#r43478027>.
>